test/applyMiddlewareTests.ts

```diff
@@ -60,12 +60,38 @@ describe("applyMiddleware", () => {
                 passedArguments = args;
             });
 
-        dispatchWithMiddleware(originalAction, originalActionType, originalArguments);
+        dispatchWithMiddleware(originalAction, originalActionType, originalArguments, null);
         expect(passedAction).toBe(originalAction);
         expect(passedActionType).toBe(originalActionType);
         expect(passedArguments).toBe(originalArguments);
     });
 
+    it("Passes action parameters to middleware with options", () => {
```

Comment:
Why don't you just modify the previous test case to validate the options too, instead of duplicating most of the test case here.